### DISCUSSION TITLE
ScriptReference/AssetDatabase.GetAssetOrScenePath

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#217] [Notes]
# ScriptReference/AssetDatabase.GetAssetOrScenePath

https://docs.unity3d.com/ScriptReference/AssetDatabase.GetAssetOrScenePath.html

**Comment by mvi** [comment]
> When a game object is part of a Scene, the Scene path is returned.

  
For clarity, the Scene path is the AssetDatabase path to the Scene, e.g. Assets/Scenes/MyScene.unity, not the hierarchy path within the scene.
